posthog/models/session_recording_event/sql.py

```diff
@@ -116,8 +120,8 @@
 
 INSERT_SESSION_RECORDING_EVENT_SQL = (
     lambda: f"""
-INSERT INTO {SESSION_RECORDING_EVENTS_DATA_TABLE()} (uuid, timestamp, team_id, distinct_id, session_id, window_id, snapshot_data, created_at, _timestamp, _offset)
-SELECT %(uuid)s, %(timestamp)s, %(team_id)s, %(distinct_id)s, %(session_id)s, %(window_id)s, %(snapshot_data)s, %(created_at)s, now(), 0
+INSERT INTO {SESSION_RECORDING_EVENTS_DATA_TABLE()} (uuid, timestamp, team_id, distinct_id, session_id, window_id, snapshot_data, events_summary, created_at, _timestamp, _offset)
+SELECT %(uuid)s, %(timestamp)s, %(team_id)s, %(distinct_id)s, %(session_id)s, %(window_id)s, %(snapshot_data)s, %(events_summary)s, %(created_at)s, now(), 0
```

Comment:
I don't think we need this now that snapshot_data contains the event_summary

Reply:
True. If we merge the write path ASAP then we might be able to figure out a point in time where we load everything via event_summary alone and then fallback to a different query for much older items. The migration here is gonna be interesting...